Libraries/Vibration/Vibration.js

```diff
@@ -27,11 +27,31 @@ var Platform = require('Platform');
  */
 
 var Vibration = {
-  vibrate: function(duration: number = 400) {
+  vibrate: function(pattern: number | Array<number> = 400, repeat: number = -1) {
```

Comment:
Why is the value of repeat -1? What happens when you set it to 0? Looks kinda weird to repeat something for -1 times.